MicroblogSpring/src/main/java/com/theironyard/controllers/MicroblogSpringController.java

```diff
@@ -0,0 +1,59 @@
+package com.theironyard.controllers;
+
+import com.theironyard.models.Message;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+
+/**
+ * Created by sparatan117 on 12/27/16.
+ */
+@Controller
+public class MicroblogSpringController {
+    public static final String SESSION_USER_NAME = "username";
+    static ArrayList<Message> messages = new ArrayList<>();
+
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model, HttpSession session) {
+       String username = (String)session.getAttribute(SESSION_USER_NAME);
+       model.addAttribute("name", username);
+       model.addAttribute("messages", messages);
+       return "home";
+    }
+
+    @RequestMapping(path = "/login", method = RequestMethod.POST)
```

Comment:
You need to start adding comments to your code.  simply type `/**` on the blank line above and hit enter to get the template generated.